spec/ReadPreferenceOption.spec.js

```diff
@@ -0,0 +1,978 @@
+'use strict'
+
+const Parse = require('parse/node');
+const ReadPreference = require('mongodb').ReadPreference;
+const rp = require('request-promise');
+
+describe('Read preference option', () => {
```

Comment:
should be a `describe_only_db('mongo')`, not sure it applies to PG yet. That will alleviate confusion.